oauth2_http/java/com/google/auth/oauth2/OAuth2Credentials.java

```diff
@@ -151,6 +151,16 @@ public final AccessToken getAccessToken() {
     return null;
   }
 
+  /** Returns the credentials' refresh margin. */
```

Comment:
We don't need those, except for testing? VisibleForTesting attribute for variables is preferable. We better limit the public surface unless required.